### ISSUE TITLE
Burn Error

### ISSUE THREAD

**Issue by georges-ph** [CTurt/FreeDVDBoot#291]
I'm still at the second step. I downloaded the ISO and used FreeISOBurner with 4x the lowest speed. It stopped at 19% and gave me an error. I bought another disc to try again with CDBurnerXP with options mentioned in the docs and stopped at 33% and gave an error.
The disc specs are Verbatim DVD-R 4.7 GB 16x speed and 120 min.
Am I missing something?

**Comment by NewRoseCustoms** [comment]
Try dragging the files onto the disc, that’s the only way I could get it to work. 
I was able to drag and drop onto an already burned disc, then you burn after files are on the disc. If you have a new disc it would be worth trying on that if the first way doesn’t work. But burning after dragging the files on seemed to be the trick, instead of directly burning the files to the disc.  I also used windows 10 default burn settings doing it that way.
